### PR TITLE
DOC-4280: Document the properties of sort in People directory

### PR DESCRIPTION
**Pull request by mzorai**

### CODE REVIEW

**Comment by mzorai** [comment]
The content of the PR could be viewable through this link: https://docs-preprod.exoplatform.org/en/preproduction/Configuration.html#configuration-sortfield

File: docs/Configuration.rst [hunk 1]
```diff
@@ -22,6 +22,9 @@ Configuration
     -  :ref:`User inactivity delay configuration <UserInactivityDelay>`
        How to define the user inactivity delay.
 
+    -  :ref:`Sort field configuration <Configuration.sortField>`
+       How to define the on which make the sort in :ref:`People directory <People_Directory>`.
```
Comment (plamarque) [inline docs/Configuration.rst]:
```suggestion
       How to define sort order in :ref:`People directory <People_Directory>`.
```

File: docs/Configuration.rst [hunk 2]
```diff
@@ -1653,6 +1656,36 @@ The parameter is expressed in millisecond and the value default is
      # The delay when we consider a user as offline. Default value is 240000 milliseconds
     exo.user.status.offline.delay=240000
     
+.. _Configuration.sortField:    
+    
+=========================    
+Sort field configuration 
+=========================
+
+In :ref:`people application <People_Directory>` users are sorted to facilitate
```
Comment (plamarque) [inline docs/Configuration.rst]:
```suggestion
In :ref:`people application <People_Directory>`, users are sorted to facilitate 
```

File: docs/Configuration.rst [hunk 3]
```diff
@@ -1653,6 +1656,36 @@ The parameter is expressed in millisecond and the value default is
...
     
+.. _Configuration.sortField:    
+    
+=========================    
+Sort field configuration 
+=========================
+
+In :ref:`people application <People_Directory>` users are sorted to facilitate 
+the search in the directory.
+It is possible to choose the field on which to make the sort by defining this
```
Comment (plamarque) [inline docs/Configuration.rst]:
```suggestion
It is possible to choose the field on which to sort by defining this 
```

File: docs/Configuration.rst [hunk 4]
```diff
@@ -1653,6 +1656,36 @@ The parameter is expressed in millisecond and the value default is
...
+In :ref:`people application <People_Directory>` users are sorted to facilitate 
+the search in the directory.
+It is possible to choose the field on which to make the sort by defining this 
+parameter in :ref:`exo.properties <Configuration.ConfigurationOverview>` file:
+
+::
+
+	exo.social.identity.sort.field=fullname
+	
+.. note:: The sort is feasible on these fields: fullname, lastname or firstname.
```
Comment (plamarque) [inline docs/Configuration.rst]:
```suggestion
.. note:: The following values are accepted: fullname, lastname or firstname.
```

Comment (plamarque) [reply docs/Configuration.rst]: I think we should explain where the full name is picked from. How different is it form what is displayed ?

File: docs/Configuration.rst [hunk 5]
```diff
@@ -1653,6 +1656,36 @@ The parameter is expressed in millisecond and the value default is
...
+
+You can also define the sort direction; ASC or DESC by defining this parameter
```
Comment (plamarque) [inline docs/Configuration.rst]:
```suggestion
You can also define the sort direction (ASC for ascending or DESC for descending) with this parameter 
```

File: docs/Configuration.rst [hunk 6]
```diff
@@ -1653,6 +1656,36 @@ The parameter is expressed in millisecond and the value default is
...
+
+You can also define the sort direction; ASC or DESC by defining this parameter 
+in :ref:`exo.properties <Configuration.ConfigurationOverview>` file:
+
+:: 
+
+	exo.social.identity.sort.direction=asc
+	
+By default, users are sorted in the :ref:`people directory <People_Directory>`
+by their fullnames with ascending order.
```
Comment (plamarque) [inline docs/Configuration.rst]:
```suggestion
by their full names in ascending order.
```

**Review by plamarque — CHANGES_REQUESTED** [review]
mainly typos and language

File: docs/Configuration.rst (hunk 4 above)
Comment (mzorai) [reply docs/Configuration.rst]: done

**Review by plamarque — APPROVED** [review]
LGTM

**Pull request merged**